feature/aboute.scss

831230 opened the pull request:
ukończone style dla sekcji about wraz z media 480, 768 i 1280

831230 committed:
feature/aboute.scss

ukończone style dla sekcji about wraz z media 480, 768 i 1280

dawidmarek95x reviewed (commented):
**W pliku about.html:**
Linijka 41: 
Nie wiem jak pobierałeś ikonkę, ale u na makiecie ma wymiary `width="4px"` i `height="8x"`. Tak bym zmienił.

**W pliku _about.scss:**
Linijki 1-7: 
Porządny `@mixin` dla właściwości `background-image` - zmienił bym jedynie nazwę na `about_bg_properties`, żeby bardziej nakierowywała do czego ją wykorzystujemy i  `background-position` na `center bottom;`. Ale ogólnie super 🥇
Jeśli zmienisz nazwę` @mixin` to poniżej w linijkach 10, 17, 104 i 113 też zmień na tą samą nazwę.

Linijki 8 - 11:
Dla klasy `.about` brakuje mi jeszcze `padding: 125px 0 39px 0;` lub `padding-top: 125px;` i `padding-bottom: 39px;`, aby były te wymagane odstępy. 
U nas w projekcie nie ma klasy section, ponieważ każda sekcja ma inne paddingi, więc musimy dodawać tak jak opisałem powyżej dla każdej osobno.

Linijki 21-26: 
Teoretycznie każdy element wewnątrz `.about__content` jest blokowy, więc nie trzeba było robić tego poprzez `display: flex;` z kierunkiem `flex-direction: column;`. Nie jest błędem :)

Linijki 28 - 31: Ponieważ masz dodaną dla `.about__content` właściwość `align-items: center;` to żeby tekst był wyrównany do lewej jak w makiecie musisz dodać dla `.about__descriptions`  lub każdemu opisowi tj. `.about__description-upper` i `.about__description-lower`właściwość `text-align: left; `.

Linijki 33 - 37:
Tutaj nie wiem czy zauważyłeś, ale jest błąd makiety, bo ostatnia linijka tekstu nie mieści się w ramce. `margin-bottom` jest więc nieco oszukany, dlatego robiąc proste obliczenia tj. 19px (line-height) * 5 (tyle jest wierszy na makiecie) to nam daje 95px wysokości całego bloku. Jak sobie rozciągniesz ten blok na makiecie to wtedy `margin-bottom: 31px;`. Taki bym ustawił :)

Linijki 46 - 60:
W linijce 55 zmień na `transition: background-color $mainTransition;`, ponieważ Ania dodała nam zmienną do przejść :)

Linijki 66 - 70:
Tak jak w pliku about.html rozmiar ikony strzałki na makiecie pokazuje `width: 4px;` i `height: 8px;` tak też bym zmienił.

Linijki 73 - 76:
Dla klasy `.about` brakuje mi jeszcze `padding: 130px 0 66px 0;` lub `padding-top: 130px;` i `padding-bottom: 66px;`, aby były te wymagane odstępy.

Linijki 78 - 83:
`display: flex;` jest niepotrzebne, bo przy 480px też go masz (powtarzanie kodu)
Na makiecie padding-top: 50px; zamiast 54px jak jest u Ciebie. _Liczę od najdłuższego elementu czyli obrazu z krową do tytułu._

Linijki 85 - 87:
Brakuje jeszcze właściwości `padding-top: 50px;` dla `.about__descriptions`, bo zauważ, że `<div>` z opisami nie zaczyna się równo z krową :)

Linijki 89 - 92:
Tutaj taka sama sytuacja co przy mobilce, że jest błąd w oznaczeniu wymiaru kontenera górnego opisu, bo ostatnia linijka tekstu nie mieści się w ramce. `margin-bottom` jest więc nieco oszukany, dlatego robiąc proste obliczenia tj. 25px (line-height) * 4 (tyle jest wierszy na makiecie) to nam daje 100px wysokości całego bloku. Jak sobie rozciągniesz ten blok na makiecie to wtedy `margin-bottom: 28px;`. Taki bym ustawił :)

Linijki 102 - 106:
Dla klasy `.about` brakuje mi jeszcze `padding: 125px 0 39px 0;` lub `padding-top: 125px;` i `padding-bottom: 39px;`, aby były te wymagane odstępy. 
Jeśli zmienisz w `@mixin` `background-position` na `center bottom;` to tutaj usuń tą właściwość.

Linijki 117 - 119: 
Zmień `padding-top: 60px;` na `padding-top: 55px;`, bo tak jest na makiecie (liczone od najdłuższego elementu czyli krowy)

831230 committed:
feature/aboute.scss

rozmiar ikony: nie wiem dlaczego ale jak zrobiłem wg rozmiarów z makiety to optycznie ewidentnie ikona była mniejsza niż w makiecie. Może zależy to od przeglądarki. Faktycznie niech będzie jak w makiecie.

wiem że Ania dodała zmienną do transition ale nie mam jej w variables - pewnie jeszcze jest nie zaimportowana. Zmieniam tak jak napisałeś i pozostawiam zakomentowane bo inaczej wywala błąd bo VSC nie widzi tej zmiennej.

Rozmiary ikonki w przycisku usunąłem całkiem z scss bo w sumie jak są w html to chyba w scss się nie daje.

831230 committed:
feature/about.scss

wymiary ikonki w button usunięte z html a dodane do scss
